notes/context-toolbar-changes.md

```diff
@@ -0,0 +1,73 @@
+Changes made
```

Comment:
Is this file being merged, or is it just to make things easier for people reviewing?

Reply:
Whichever you prefer.